src/redux/rootSaga.js

```diff
@@ -0,0 +1,55 @@
+import axios from 'axios';
+import {
+  takeLatest,
+  call,
+  put,
+  select,
+} from 'redux-saga/effects';
+import {
+  types,
+  getDataSuccess,
+  getDataFailure,
+} from './reducers/data';
+
+const getState = (state, slice) => state[slice];
+
+// For socrata only
+// const dataResources = {
+//   2019: 'pvft-t768',
+//   2018: 'h65r-yf5i',
+//   2017: 'd4vt-q4t5',
+//   2016: 'ndkd-k878',
+//   2015: 'ms7h-a45h',
+// };
+
+function* getData() {
+  const {
+    startDate,
+    endDate,
+    councils,
+    requestTypes,
+  } = yield select(getState, 'data');
+
+  const endPoint = 'http://ec2-3-84-149-45.compute-1.amazonaws.com/pins';
```

Comment:
AAAAAHHHHHHHHHHHHH
can we move this to an environment variable pllzzzz
#305

Reply:
fixed